src/pages/ProfilePage.js

```diff
@@ -23,7 +29,14 @@ export const ProfilePage = () => {
               <strong className="m-r-1">Email:</strong>
               {user?.email}
             </li>
-            <button className="btn primary-btn centered-text w-95p m-h-1" onClick={logoutHandler}>
+            <button
+              className="btn primary-btn centered-text w-95p m-h-1"
+              onClick={() => {
```

Comment:
Just a suggestion - you may shift these functions inside a handler. That will look cleaner.